docs/admin-manual/ep-policy-configuration.rst

```diff
@@ -3669,6 +3664,25 @@ memory limit:
     
 Note: :ad-attr:`Memory` is in MB, thus it needs to be scaled to bytes.
 
+Docker Image Management
+'''''''''''''''''''''''
+
+A Docker container must be run from a Docker image stored in the local docker image cache.
+HTCondor will automatically fetch the required image if it is not already in the local
+image cache of the Execution point when the job starts.  HTCondor will not remove this
+image from the cache when the job leaves the EP, in order to re-use the image and not
+require a remote fetch from some Docker hub the next time a job with this image runs
+on the same Execution Point.  However, in order to not fill up the local disk that holds
+the image cache, it will periodically delete older docker images in a least-recently-used
+method, trying to keep :macro:`DOCKER_IMAGE_CACHE_SIZE` images available.  The default
+value is 8.  HTCondor will only remove images from the docker cache that HTCondor
+has placed there.  To keep track of which images it owns, as of HTCondor 24.7,
+it tags every image in the cache with an tag name that begins with "htcondor.org".
+While this may look like there are twice the images in the cache, these tags act like
+hard links, and do not consume additional space.  Note that HTCondor will not share
+docker images across users, because some images may be protected, and HTCondor cannot
+prove that one user should have access to another user's image.
+
```

Comment:
```suggestion
hard links, and do not consume additional space.

.. note::
    HTCondor will not share
    docker images across users, because some images may be protected, and HTCondor cannot
    prove that one user should have access to another user's image.

```